pykafka/cluster.py

```diff
@@ -234,10 +238,14 @@ def _request_metadata(self, broker_connects, topics):
                                     self._offsets_channel_socket_timeout_ms,
                                     buffer_size=1024 * 1024,
                                     source_host=self._source_host,
-                                    source_port=self._source_port)
+                                    source_port=self._source_port,
+                                    ssl_config=self._ssl_config)
                     response = broker.request_metadata(topics)
                     if response is not None:
                         return response
+                # FIXME: handle ssl errors separately somehow, because if we
+                # don't, _get_metadata() might attempt to treat it as if it
+                # were a zookeeper host, which gets messy
```

Comment:
I was thinking about this a bit, and strictly speaking, it is not a regression caused by this pull request - in the sense that if you use current master and try to connect to an ssl port (which obviously won't work) you'd already get stuck in the same way. Also, it's mostly just a nuisance, in that if the user gets the config wrong, the feedback is unhelpful.

So how about I remove these "fixme" lines here, but cross-reference them on #452 as an additional motivation for it, and we could still keep #452 on hold for the next version bump?